django/src/rdwatch/tasks/__init__.py

```diff
@@ -438,14 +438,15 @@ def delete_temp_model_runs_task() -> None:
         ):
             ModelRun.objects.filter(pk__in=model_runs).delete()
 
-
-@shared_task
-def delete_export_files() -> None:
-    """Delete all S3 Export Files that are over an hour old."""
-    exports_to_delete = AnnotationExport.objects.filter(
+    # Delete all S3 Export Files that are over an hour old
+    AnnotationExport.objects.filter(
         created__lte=timezone.now() - timedelta(hours=1)
-    )
-    exports_to_delete.delete()
+    ).delete()
+
+    # Delete all SatelliteFetching tasks that are over an day old
+    SatelliteFetching.objects.filter(
+        timestamp__lte=timezone.now() - timedelta(days=1)
```

Comment:
may want a status filter on this so they are not in the status of "Running", I.E. they are Complete or Error.  Just so if we have some insane long-running download tasks that are in status running it doesn't get removed for no reason.  It's a rare corner case but could happen.

Reply:
I was hesitant to add this because a task that dies to to a bug for example, may never set its SatelliteFetching to Complete or Error. But I think I found a good alernative, I changed it to check the associated celery `TaskResult` status, and if that is still `Running` then it doesn't get deleted.

Reply:
perfect, that's a better implementation.